### PR TITLE
Move personal links into about me

### PR DESCRIPTION
**Pull request by mollygraton**
Also changing background color of the site.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,35 +9,36 @@
...
-        <a href="https://www.github.com/mollygraton/" target="_blank">GitHub</a>
-    </div>
-    <div id="aboutme"> 
         <p> Hi! My name is Molly and I'm a rising sophomore at UIUC studying
             computer science. Apart from academics, I love to skateboard and
-            play ukulele! Welcome to my portfolio!
+            play ukulele! Welcome to my portfolio! Check out my 
+            <a href="https://www.linkedin.com/in/mollygraton" target="_blank">LinkedIn</a>
+            or <a href="https://www.github.com/mollygraton/" target="_blank">GitHub</a>
+            for more info.
```
Comment (cisxingg) [inline portfolio/src/main/webapp/index.html]: Though this looks super clean and nice, the style guide actually wants us to use a 2-space indentation:
https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
 body {
-    background-color: lightsalmon;
+    background-color:lightcyan;
+}
+
+#academics {
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+    width: 80vw;
```
Comment (cisxingg) [inline portfolio/src/main/webapp/style.css]: Same nit as above :)

**Review by cisxingg — CHANGES_REQUESTED** [review]
LG in general, just need to fix a couple styling issues.

**Review by cisxingg — APPROVED** [review]
LG, get reviews from Warfa please before merging :)

**Review by warfa-google — APPROVED** [review]
LGTM

**Review by warfa-google — APPROVED** [review]

**Pull request merged**